app/src/main/res/values/strings.xml

```diff
@@ -255,5 +255,9 @@
     <string name="tell_me_more">Tell me more</string>
     <string name="open_youtube_external_tabs_description">Enable to open Youtube links on external app</string>
     <string name="pref_title__open_youtube_external">Youtube links</string>
+    <string name="pref_desc__sub_theme">Change the theme of your channel</string>
```

Comment:
I think channel is hubzilla specific, in d* not, but maybe profile or account (which should fit for both)